apps/admin-portal/src/components/user-stores/wizards/basic-details-user-store.tsx

```diff
@@ -16,11 +16,12 @@
 * under the License.
 */
 
-import React, { useState, useEffect } from "react";
-import { Forms, FormValue, Field } from "@wso2is/forms";
-import { TypeResponse, AlertLevels } from "../../../models";
-import { getTypes } from "../../../api";
+import { AlertLevels, TypeResponse } from "../../../models";
+import { Field, Forms, FormValue } from "@wso2is/forms";
+import React, { useEffect, useState } from "react";
```

Comment:
Let's get this import to the top

Reply:
Eslint doesn't allow it. Imports have to be alphabetically sorted.